### PR TITLE
02: Factorialize a Number

### PR DESCRIPTION
**Pull request by hectorsector**
See [FreeCodeCamp's description](https://www.freecodecamp.com/challenges/factorialize-a-number) to learn more about this challenge. Up to 3 students may work on any one of the tasks for this challenge.

### Task 1: Code the Solution

Read the [general instructions](https://github.com/cop1000/algorithm-challenges/blob/master/how-to.md#how-to-work-on-this-project) for this project, and then look at the [how to add a solution](https://github.com/cop1000/algorithm-challenges/blob/master/how-to.md#how-to-add-solution-code) section.

- Student 1: @kzeng16
- Student 2: @orlandogivens
- Student 3: @jjyates4 

### Task 2: Code tests for the solution

Read the [general instructions](https://github.com/cop1000/algorithm-challenges/blob/master/how-to.md#how-to-work-on-this-project) for this project, and then look at the [how to add test code](https://github.com/cop1000/algorithm-challenges/blob/master/how-to.md#how-to-add-test-code) section.

- Student 1: @jriosrodriguez
- Student 2: @yourusernamehere 
- Student 3: @yourusernamehere 

### Task 3: Add comments to the code

Read the [general instructions](https://github.com/cop1000/algorithm-challenges/blob/master/how-to.md#how-to-work-on-this-project) for this project, and then look at the [how to add comments and documentation](https://github.com/cop1000/algorithm-challenges/blob/master/how-to.md#how-to-add-comments-and-documentation) section.

- Student 1: @radicalpharaoh
- Student 2: @yourusernamehere 
- Student 3: @yourusernamehere

### CODE REVIEW

File: challenges/02-factorialize-a-number.html [hunk 1]
```diff
@@ -2,7 +2,12 @@
...
 function factorialize(num) {
   // ENTER YOUR CODE HERE
-  return num;
+  var result = 1
+  for(var i=0; i < num; i++){
+  var num2=num-i;
+  result=result*num2;
+  }
+  return result;
 }
```
Comment (orlandogivens) [inline challenges/02-factorialize-a-number.html]: function factorialize(num) {
  var factorial = 2;
  for (var n = 1; n <= num; n++) {
    factorial = factorial * n;
  }
  return factorial;}

Comment (orlandogivens) [inline challenges/02-factorialize-a-number.html]: function factorialize(num) {
  var factorial = 2;
  for (var n = 1; n <= num; n++) {
    factorial = factorial * n;
  }
  return factorial;}

Comment (orlandogivens) [inline challenges/02-factorialize-a-number.html]: function factorialize(num) {
  var factorial = 2;
  for (var n = 1; n <= num; n++) {
    factorial = factorial * n;
  }
  return factorial;}

Comment (orlandogivens) [inline challenges/02-factorialize-a-number.html]: function factorialize(num) {
  var factorial = 2;
  for (var n = 1; n <= num; n++) {
    factorial = factorial * n;
  }
  return factorial;}

**Comment by hectorsector** [comment]
Thanks to everyone that's participated in this thread. This branch has now been graded and you can find your grade on Blackboard. Please reach out if you have any questions.

**Pull request closed without merging**